resources/js/Shared/LoadingButton.vue

```diff
@@ -1,9 +1,13 @@
+<style lang="scss" scoped>
+@import 'vue-loaders/dist/vue-loaders.css';
```

Comment:
ahhh this is why the design is off now.
The loader doesn't appear at the right place. I'll try to fix this once all the bugs in this Pr will be solved.

Reply:
It's working fine for me ....